setup/config/config_clientapi.go

```diff
@@ -74,34 +74,45 @@ func (c *ClientAPI) Defaults(opts DefaultOpts) {
 func (c *ClientAPI) Verify(configErrs *ConfigErrors) {
 	c.TURN.Verify(configErrs)
 	c.RateLimiting.Verify(configErrs)
-	if c.RecaptchaEnabled {
-		if c.RecaptchaSiteVerifyAPI == "" {
-			c.RecaptchaSiteVerifyAPI = "https://www.google.com/recaptcha/api/siteverify"
-		}
-		if c.RecaptchaApiJsUrl == "" {
-			c.RecaptchaApiJsUrl = "https://www.google.com/recaptcha/api.js"
+
+	if c.MSCs.MSC3861Enabled() {
+		if c.RecaptchaEnabled || !c.RegistrationDisabled {
```

Comment:
Maybe just me, but I prefer reading the "not" path first.

```suggestion
		if !c.RegistrationDisabled || c.RecaptchaEnabled {
```

For a different PR: We should only have either `Enabled` or `Disabled` in our config. Otherwise stuff like this might become confusing.

Reply:
I don't have any preferences regarding the "not" path, but I agree that registration should go first. :smile: